ttnn/cpp/ttnn/operations/core/core.cpp

```diff
@@ -13,7 +13,7 @@
 
 namespace ttnn::operations::core {
 
-ttnn::Tensor reshape(const ttnn::Tensor& tensor, const ttnn::Shape& shape) {
+ttnn::Tensor reshape(const ttnn::Tensor& tensor, const ttnn::Shape& shape, const std::optional<MemoryConfig>& memory_config) {
```

Comment:
This arg seems unused for this fn so why do we need it added here?

Reply:
No reason for it that I can see.